React/Fabric/RCTSurfacePresenter.mm

```diff
@@ -82,7 +82,6 @@ @implementation RCTSurfacePresenter {
   butter::shared_mutex _observerListMutex;
   NSMutableArray<id<RCTSurfacePresenterObserver>> *_observers;
   RCTImageLoader *_imageLoader;
-  RuntimeExecutor _runtimeExecutor;
 }
```

Comment:
Must have been a merge conflict when merging 0.61 and 0.63
https://github.com/microsoft/react-native-macos/blame/main/React/Fabric/RCTSurfacePresenter.mm

Reply:
It's also on line 81?

Reply:
FWIW, I find this link useful for comparing our diffs with RN Core. I set the RN Core branch in the diff to 0.68-stable since we're currently synced to that.

https://github.com/microsoft/react-native-macos/compare/main...facebook:react-native:0.68-stable

Reply:
That is a handy tool ;-)

In facebook/react-native only the first definition exists
https://github.com/facebook/react-native/blob/main/React/Fabric/RCTSurfacePresenter.mm#L74-L87